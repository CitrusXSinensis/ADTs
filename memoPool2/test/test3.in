ra1x
pa
ca
aa 3
pa
ca
aa 4
aa 3
aa 5
pa
ca
fa 6
pa
ca
fa 4
pa
ca
ra1 3 4 5 6x
pa
ca

