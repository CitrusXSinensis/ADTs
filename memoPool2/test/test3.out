1
1/4
1 3
2/4
1 3 4 3 5
5/8
Nullptr
1 3 4 3 5
5/8
Chunk: 4
1 3 3 5
4/8
1 3 4 5 6
5/8
